### PR TITLE
Make it ready for transfer to infinispan organisation

### PR DESCRIPTION
**Pull request by blocha**
Changed versions to last ones, changed packages names, deleted sample-project.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,10 +30,8 @@ hs_err_pid*
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
 # User-specific stuff:
-.idea/workspace.xml
 .idea/tasks.xml
```
Comment (slaskawi) [inline .gitignore]: All those files can be deleted. I see you added `.idea/` at the end of the file.

File: infinispan-sample-app/pom.xml [hunk 2]
```diff
@@ -1,51 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
```
Comment (slaskawi) [inline infinispan-sample-app/pom.xml]: Could you please create a sample app in [the tutorials](https://github.com/infinispan/infinispan-simple-tutorials). Of course it won't compile now but once we release the starters it should work fine.

Comment (slaskawi) [inline infinispan-sample-app/pom.xml]: I also think we should add some information in a READ.ME file how to use this..

**Comment by slaskawi** [comment]
@galderz @gustavonalle May I ask you for review?

**Comment by blocha** [comment]
Ok, I changed gitignore and added some more informations to readme. Hope you like it :)

**Comment by blocha** [comment]
Added pull request for example project: https://github.com/infinispan/infinispan-simple-tutorials/pull/16

**Comment by slaskawi** [comment]
Integrated, thanks @blocha !

**Pull request merged**